### PR TITLE
Add Input for KPI calcualtions

### PR DESCRIPTION
**Pull request by JuergenSoeren**
Includes first mapping on financial modeling prep api

### CODE REVIEW

**Comment by JuergenSoeren** [comment]
Guckt mal ob das verständlich ist..

File: README.md [hunk 1]
```diff
@@ -16,6 +16,30 @@ Here can the Firebase functions be found that are necessary to successfully setu
...
+
+##
+
+
+## Financials Input required annually for Big Five (Last 10 Years)
+
+|Name (DE)|Source|Name (EN)|Finacial Modeling Prep API| Comments |
+|---|---|---|---|---|
+|Summe langfristiges Fremdkapital| Bilanz (Passiva)  | Balance Sheet > Long-term Debt  | https://financialmodelingprep.com/api/v3/financials/balance-sheet-statement/AAPL  |   |
+|Summe Eigenkapital   | Bilanz (Passiva)  | Balance Sheet > Total shareholders equity  |   |   |
```
Comment (arneous) [inline README.md]: Also found this here: https://financialmodelingprep.com/api/v3/financials/balance-sheet-statement/AAPL

File: README.md [hunk 2]
```diff
@@ -16,6 +16,30 @@ Here can the Firebase functions be found that are necessary to successfully setu
...
+|Summe Passiva   | Bilanz (Passiva)  | Balance Sheet > Total liabilities  |   |   |
+|Umsatz   | GuV  | Income Statement > Revenue  | https://financialmodelingprep.com/api/v3/financials/income-statement/AAPL  |   |
+|Operatives Ergebnis   | GuV  | Income Statement > Operating Income  |   |   |
+|Steuern vom Einkommen und vom Ertrag   | GuV | Income Statement > Income Tax Expense  |   |   |
+|Jahresüberschuss | GuV  | Income Statement > Gross Profit  |   |   |
+|Cash flow   | Cashflow Statement  | Net cash flow | https://financialmodelingprep.com/api/v3/financials/cash-flow-statement/AAPL  |   |
+|Cashflow aus der Investitionstätigkeit   | Cashflow Statement | Investing Cash Flow  |   |   |
+|Ausstehende Aktien in Mio. (verwässert)  | "Wertpapierdaten"  |  Income Statement > Weighted Average Shs Out | https://financialmodelingprep.com/api/v3/financials/income-statement/AAPL  |   |
+|Ergebnis je Aktie verwässert   | "Wertpapierdaten"  |   |   | Kann man auch berechnen  |
```
Comment (arneous) [inline README.md:41]: Is das nicht "EPS Diluted" in https://financialmodelingprep.com/api/v3/financials/income-statement/AAPL ?

File: README.md [hunk 3]
```diff
@@ -16,6 +16,30 @@ Here can the Firebase functions be found that are necessary to successfully setu
...
+|Umsatz je Aktie   | "Wertpapierdaten"  |   |   | Kann man auch berechnen |
```
Comment (arneous) [inline README.md:42]: Vielleicht "Revenue per Share" aus https://financialmodelingprep.com/api/v3/company-key-metrics/AAPL

**Review by arneous — COMMENTED** [review]
Updated list

**Comment by arneous** [comment]
I did a review!

**Review by arneous — CHANGES_REQUESTED** [review]
Lets also check other fields in
https://financialmodelingprep.com/api/v3/company-key-metrics/AAPL

**Pull request merged**